torch/csrc/autograd/functions/errors.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PYTORCH_AUTOGRAD_ERRORS_H
+#define PYTORCH_AUTOGRAD_ERRORS_H
+
+#define PT_ERR_BACKWARD_TWICE "Trying to backward through the " \
```

Comment:
Huh? What two libraries? Also, such symbol conflict is harmless because the data is the same in both libraries.